app/models/audit/AuditTaskTable.scala

```diff
@@ -506,136 +509,47 @@ object AuditTaskTable {
   }
 
   /**
-    * Get tasks in the region
+    * Get tasks in the region. Called when an anonymous user begins auditing a region.
     *
     * @param regionId Region id
     * @return
     */
-  def selectCompletedTasksInARegion(regionId: Int, userId: UUID): List[NewTask] = db.withSession { implicit session =>
+  def selectTasksInARegion(regionId: Int): List[NewTask] = db.withSession { implicit session =>
     val timestamp: Timestamp = new Timestamp(Calendar.getInstance(TimeZone.getTimeZone("UTC")).getTime.getTime)
```

Comment:
Okay the diff is a incorrect here. What I actually did was remove the `selectCompletedTasksInARegion` function (which is not used anywhere), and I rewrote both the `selectTasksInARegion(regionId)` and `selectTasksInARegion(regionId, user)` functions.

The diff makes it look like I rewrote `selectCompletedTasksInARegion` (and renamed it `selectTasksInARegion(regionId)`), rewrote `selectTasksInARegion(regionId)` (and renamed it `selectTasksInARegion(regionId, user)`), and deleted `selectTasksInARegion(regionId, user)`...

For the functions I rewrote, I modified them to use Slick, and mostly just to reuse the code that had been written previously. So I am using `nonDeletedStreetEdgeRegions` and `streetCompletionCounts`, which were defined in other files, and have multiple joins, etc.